project-p2-layouts-master/code/style.css

```diff
@@ -0,0 +1,252 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;500;600;900&display=swap');
+
+body {
+  background-color:	rgb(177, 212, 195);
+}
+
+img {
+  width: 60%;
+  border-radius: 50%;
+}
+.center {
+  display: block;
+  margin-left:auto;
+  margin-right: auto; 
+  width: 50%; 
+}
+#mySidenav a {
+  position: absolute; 
+  left: -40px; 
+  transition: 0.3s; 
+  padding: 40px; 
+  margin: 50px; 
+  width: 50px; 
+  height: 50px;
+  text-decoration: none; 
+  font-size: 14px; 
+  font-family: 'Roboto Slab', serif ;
+  font-weight: bolder;
+  color: rgb(249, 250, 249); 
+  border-radius: 50%; 
+  text-align: center;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+}
+
+#mySidenav a:hover {
+  left: 0; 
+}
+#HOME {
+  top: 0px;
+  background-color:rgb(205, 241, 229);
+  margin-top: 120px; 
+  margin-bottom: 120px;
+}
+
+#IDEE {
+  top: 150px;
+  background-color: rgb(162, 216, 197);
+  margin-top: 120px;
+  margin-bottom: 120px;
+}
+
+#FRAGEBOGEN {
+  top: 300px;
+  background-color:rgb(140, 186, 172);
+  margin-top: 150px;
+  margin-bottom: 120px; 
+  margin-left: 2px;
+}
+
+#ÜBER {
+  top: 450px;
+  background-color:	rgb(124, 180, 156); 
+  margin-top: 120px;
+  margin-bottom: 120px;
+}
+
+#PORTFOLIO{
+  top: 600px; 
+  background-color: rgb(34, 90, 71);
+  margin-top: 120px; 
+  margin-bottom: 120px; 
+  font-size: 13px; 
+}
+
+h2 {color:rgb(108, 164, 140);
+font-family: 'Roboto Slab', serif; 
+font-size: 90px;
+text-align: center;
+margin-top: 55px;
+margin-bottom: 5px;
+padding-left: 30px;
+}
+
+p {color: rgb(108, 164, 140);
+  font-family: 'Roboto Slab', serif;
+  font-size: 30px;
+  text-align: center;
+  margin-bottom: 5px;
+  margin-top: 5px; 
+  padding-top: 2px;
+  padding-left: 300px;
+  padding-right: 250px;
+  padding-bottom: 20px;
+}
+
+.fa {
+padding: 10px;
+width: 20px;
+height: 20px;
+text-align: center;
+margin: 5px
+}
+
+.fa:hover {
+opacity: 0.5;
+}
+
+.fa-instagram {
+background:rgb(251, 221, 218);
+color:rgb(249, 250, 249);
+border-radius:50%; 
+font-size: 20px; 
+font-weight: bolder;
+line-height: 100%;
+}
+
+
+.link2 {
+text-align: justify;
+color: rgb(108, 164, 140)}
+
+.woman {
+  text-align: center; 
+  line-height: 20%;
+  font-size: 30px;
+  font-family: 'Roboto Slab', serif;
+  text-decoration: none;
+  color:rgb(249, 250, 249);
+  margin: 2px;  
+}
+/*I don't want the names(links) to be underlined, 
+I want them white, why doesn't it work? */
```

Comment:
You need to specify that it should apply to the links _inside_ the `div` with the class name `woman`, so:
```
.woman a {
  text-decoration: none;
  color: white;
}
```